meta/main.yml

```diff
@@ -12,6 +12,11 @@ galaxy_info:
       versions:
         - 6
         - 7
+    - name: Fedora
+      versions:
+        - 27
+        - 28
+        - 29
```

Comment:
This would also need `30` and maybe we can take off 27/28 since they are EOL'ed?

Reply:
Done